Lib/distutils/tests/test_spawn.py

```diff
@@ -124,6 +124,11 @@ def test_find_executable(self):
                     rv = find_executable(program)
                     self.assertEqual(rv, filename)
 
+    def test_spawn_missing_exe(self):
+        with self.assertRaises(DistutilsExecError) as ctx:
+            spawn(['does-not-exist'])
+        assert "command 'does-not-exist' failed" in str(ctx.exception)
```

Comment:
I would prefer to use self.assertIn(), since it produces a more helpful error message if the test fails. Otherwise, it's painful to debug on buildbots.

Reply:
ugh. Okay.

Reply:
We don't invoke pytest black magic in the Python test suite. It's the raw unittest module.

Reply:
I actually prefer native asserts because I use pytest with assert-rewrite, so `self.assertIn` is actually another level of indirection and leads to failures in the wrong stack frame. But for consistency, I'll rewrite it.

Reply:
Fixed in 1f20346.